include/seqan3/core/metafunction/basic.hpp

```diff
@@ -34,6 +34,26 @@ namespace seqan3
 template <typename t>
 using remove_cvref_t = std::remove_cv_t<std::remove_reference_t<t>>;
 
+// ----------------------------------------------------------------------------
+// remove_rvalue_reference
+// ----------------------------------------------------------------------------
+
+/*!\brief Return the input type with `&&` removed, but lvalue references preserved [Type metafunction].
+ * \tparam t The type to operate on.
+ */
```

Comment:
Please add a link to the shortcut, since they are not grouped together in the api documentation